allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
> this is not the last change to allure ever
Of course. So it would be better to make this change as small as possible to make it easier to review.